profiles/standard/standard.install

```diff
@@ -124,6 +124,23 @@ function standard_install() {
     node_add_body_field($type);
   }
 
+  $node = array(
+    'title' => 'Your first article!',
+    'body' => array(
+      LANGUAGE_NONE => array(
+        array(
+          'value' => 'This is your first article, edit it or delete it.',
```

Comment:
Let's wrap the string in `st()` here so it can be translated into user's install language. We don't have install language packages yet, but when we do it'll be nice to be ready.